test/e2e/suite/src/test/resources/cases/dql/dataset/db/select_sys_data_for_pg.xml

```diff
@@ -22,22 +22,19 @@
         <column name="Type" />
         <column name="Owner" />
     </metadata>
-    <row values="public | t_broadcast_table | table | mock user" />
-    <row values="public | t_country | table | mock user" />
-    <row values="public | t_merchant | table | mock user" />
-    <row values="public | t_order | table | mock user" />
-    <row values="public | t_order_aggregation_view | view | mock user" mayNotExist="true" />
-    <row values="public | t_order_federate | table | mock user" />
-    <row values="public | t_order_federate_sharding | table | mock user" />
-    <row values="public | t_order_item | table | mock user" />
-    <row values="public | t_order_item_federate | table | mock user" />
-    <row values="public | t_order_item_federate_sharding | table | mock user" />
-    <row values="public | t_order_item_join_view | view | mock user" mayNotExist="true" />
-    <row values="public | t_order_subquery_view | view | mock user" mayNotExist="true" />
-    <row values="public | t_order_union_view | view | mock user" mayNotExist="true" />
-    <row values="public | t_product | table | mock user" />
-    <row values="public | t_product_category | table | mock user" />
-    <row values="public | t_product_detail | table | mock user" />
-    <row values="public | t_single_table | table | mock user" />
-    <row values="public | t_user | table | mock user" />
+    <row values="public | t_broadcast_table | table | postgres" />
```

Comment:
Please support PostgreSQL \d statements with sql federation. If the current scenario is table sharding, the table name returned is incorrect.

Reply:
The content of it-env.properties is below:
"it.scenarios=db"

How can I figure out the correct table name?

Reply:
The content of it-env.properties is below:
"it.scenarios=db"

How can I figure out the correct table name?